frontend/__tests__/components/Breadcrumbs/breadcrumbs.test.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { render, screen, renderer } from "../../../test-utils";
+import Breadcrumbs from "../../../components/Breadcrumbs/Breadcrumbs";
+
+describe("Breadcrumbs", () => {
+  it("should render breadcrumbs component", () => {
+    render(
+      <Breadcrumbs
+        path={[{ id: "crumb-path", page: "crumb", href: "/to-the-crumb" }]}
```

Comment:
I don't believe you are testing all possible UI outcomes here. Look into RTL best practices.

https://infinum.com/handbook/frontend/react/testing-best-practices

Reply:
I added a test for clicking on the breadcrumb component. Hopefully, that will suffice

Reply:
A test for clicking on the component is not enough, although it is considerably better - you are not testing all possible UI scenarios of this component. Once again, please take a look at some front end testing best practices - we want to make sure we are testing every possible scenario that could happen and make sure that the behavior is what we would like to see.